user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQICoreDataSource.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: GQI_IGQICoreDataSource
+---
+
+# IGQICoreDataSource interface
+
+Available from DataMiner 10.3.3/10.4.0 onwards<!-- RN 35389 -->.
```

Comment:
I see that RN 35389 is set to soft launch. Does this mean that this is only available if a soft-launch option is enabled?

Reply:
No, it was set to soft launch because Custom Operators as a whole were still in soft launch at the time.
This was moved out of soft launch with [RN37840](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=37840).